src/styles/blocks/page.scss

```diff
@@ -0,0 +1,28 @@
+.page {
+  font-family: "Cinzel", serif;
```

Comment:
Consider using a variable for the font-family to make it reusable and easier to maintain.